tests/integration/api_v3/http/transactions.test.js

```diff
@@ -346,6 +371,39 @@ describe('Transactions API', () => {
 		});
 	});
 
+	describe('Retrieve transaction list by receivingChainID', () => {
+		it('should return transactions when called with receivingChainID', async () => {
+			if (refTransaction.params.receivingChainID) {
+				const response = await api.get(
+					`${endpoint}?receivingChainID=${refTransaction.params.receivingChainID}`,
+				);
+				expect(response).toMap(goodRequestSchema);
+				expect(response.data).toBeInstanceOf(Array);
+				expect(response.data.length).toBeGreaterThanOrEqual(1);
+				expect(response.data.length).toBeLessThanOrEqual(10);
```

Comment:
Isn't default limit 100? If so, then we should either update this check to 100 or add a limit param to the api call.

Reply:
Default limit is always `10`.